endpoints-support.md

```diff
@@ -24,6 +24,12 @@
 |<sub>/rest/connection-templates/defaultConnectionTemplate</sub>                          |GET          | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/connection-templates/{id}</sub>                                               |GET          | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
 |<sub>/rest/connection-templates/{id}</sub>                                               |PUT          | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|     **Deployment Plans**                                                                                         |
```

Comment:
Image streamer resources should be placed separately. Please refer to Ruby SDK.

Reply:
done